packages/documentation/src/stories/getting-started/getting-started.docs.mdx

```diff
@@ -0,0 +1,135 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as GettingStartedStories from './getting-started.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Getting started
+
+Welcome to the Swiss post design system world!
+
+## Which package do I need?
+
+The design-system is formed with several packages and different compatibilities with your favorite framework.
+
+<div role="alert" className="alert alert-info">
+  All components require at least the <a href="/?path=/docs/e53e2de8-0bbf-4f70-babc-074c5466f700--docs"><span className="d-inline-block text-decoration-none" aria-hidden="true">📦 </span>Styles package</a>.
+</div>
+
+<div className="table-responsive">
```

Comment:
I think it could also be a good idea to add the package tiles from the home page here instead of simple links with an font icon.
Maybe we would need a smaller version of them tiles, but it could lighten up the table a lot.